contracts/GenArt721CoreV3.sol

```diff
@@ -338,31 +344,53 @@ contract GenArt721CoreV3 is ERC721, Ownable, IGenArt721CoreContractV3 {
     }
 
     /**
-     * @notice Updates additional payee for project `_projectId` to be
-     * `_additionalPayee`, receiving `_additionalPayeePercentage` percent
-     * of artist mint and royalty revenues.
+     * @notice Updates additional payees and percentage splits for project
+     * `_projectId`.
+     * @param _projectId Project ID.
+     * @param _additionalPayeePrimarySales Address that may receive a
+     * percentage split of the artit's primary sales revenue.
+     * @param _additionalPayeePercentagePrimarySales Percent of artist's
+     * portion of primary sale revenue that will be split to address
+     * `_additionalPayeePrimarySales`.
+     * @param _additionalPayeeRoyalties Address that may receive a percentage
+     * split of the secondary sales royalties.
+     * @param _additionalPayeePercentageRoyalties Percent of artist's portion
+     * of secondary sale royalties that will be split to address
+     * `_additionalPayeeRoyalties`.
      */
-    function updateProjectAdditionalPayeeInfo(
+    function updateProjectAdditionalPayees(
         uint256 _projectId,
-        address payable _additionalPayee,
-        uint256 _additionalPayeePercentage
-    ) public onlyArtist(_projectId) {
-        require(_additionalPayeePercentage <= 100, "Max of 100%");
-        projectIdToAdditionalPayee[_projectId] = _additionalPayee;
-        projectIdToAdditionalPayeePercentage[
+        address payable _additionalPayeePrimarySales,
+        uint256 _additionalPayeePercentagePrimarySales,
+        address payable _additionalPayeeRoyalties,
+        uint256 _additionalPayeePercentageRoyalties
+    ) external onlyArtist(_projectId) {
+        projectIdToAdditionalPayeePrimarySales[
+            _projectId
+        ] = _additionalPayeePrimarySales;
+        projectIdToAdditionalPayeePrimarySalesPercentage[
             _projectId
-        ] = _additionalPayeePercentage;
+        ] = _additionalPayeePercentagePrimarySales;
+        projectIdToAdditionalPayeeRoyalties[
+            _projectId
+        ] = _additionalPayeeRoyalties;
+        projectIdToAdditionalPayeeRoyaltiesPercentage[
+            _projectId
+        ] = _additionalPayeePercentageRoyalties;
     }
 
     /**
      * @notice Updates artist secondary market royalties for project
      * `_projectId` to be `_secondMarketRoyalty` percent.
+     * This DOES NOT include the secondary market royalty percentages collected
+     * by Art Blocks; this is only the total percentage of royalties that will
+     * be split to artist and additionalSecondaryPayee.
      */
     function updateProjectSecondaryMarketRoyaltyPercentage(
         uint256 _projectId,
         uint256 _secondMarketRoyalty
     ) public onlyArtist(_projectId) {
-        require(_secondMarketRoyalty <= 100, "Max of 100%");
+        require(_secondMarketRoyalty <= 30, "Max of 30%");
```

Comment:
I feel like 30% is an overly opinionated take to make at the core-contract level. I understand the concern around 100% not making sense as it adds up to 102.5%, but would propose we may just want to make this 95% then to avoid that issue? I can see an artist in the future down the line doing something interesting with decaying royalties when on-chain royalties are supported by marketplaces (though ultimately w/ wrappers, royalties do need to be enforced socially above all else) and am not sure we want to permanently peg this at 30% max which feels somewhat arbitrary to me?

Reply:
30% is ~arbitrary, but my thoughts here were somewhere along the lines of, "as a token holder, I would like some level of guarantee that an artist won't decide to extract more than some percentage of my token's sale value while I hold the token".

That being said, I don't have any strong opinions on the best number, and I would be fine with making this 95% (especially since royalties are socially enforced at the end of the day).

I'll update to 95%, but am open to further discussion/changes at any time prior to contract deployment!

Reply:
Yeah that SGTM to be open to changing in the future, but 95% for now makes sense to me.

> 30% is ~arbitrary, but my thoughts here were somewhere along the lines of, "as a token holder, I would like some level of guarantee that an artist won't decide to extract more than some percentage of my token's sale value while I hold the token".

Definitely agree with your sentiment here but also feel more strongly that this stuff just needs to be socially enforced at the end of the day given it must be incorporated by the marketplace.

Reply:
^for what it's worth I'd feel the same way about this: I think the contract should set the absolute upper-limit, and then we can socially guide the artist & users with our policy decisions. I totally agree that the contract gives minters peace of mind but they if they're worried about secondary rates for a particular contract they could check the projectDetails() and see the hard-coded split %